yarn.lock

```diff
@@ -18530,7 +18530,7 @@ react-router@5.2.0:
     tiny-invariant "^1.0.2"
     tiny-warning "^1.0.0"
 
-react-router@6, react-router@6.3.0:
```

Comment:
I don't understand why you have this change 🤔

Reply:
Did not see that but I actually get the same change if I take the master and  run `yarn lerna clean && yarn install` with yarn 1.22.18 and node 12.22.7

Reply:
this is a nit.

This  PR can be approved